code/__DEFINES/clothing.dm

```diff
@@ -49,6 +37,49 @@
 
 #define slots_amt			20 // Keep this up to date!
 
+//I hate that this has to exist

```

Comment:
One is binary bit values, or "bitflags", the other is just raw numbers, this has to exist because our slot system is unbelievably shit, so clothing defines where it can be worn as a "bitfield" (a collection of "bitflags" stored in one number because yay binary), I have to convert the ```slot``` arg passed by ```equipped()``` to a bitflag appropriate for it's purpose (eg: slot_back to SLOT_BACK) to then see if it's in ONE of the clothing's slot flags. typically ```equipped()```s are written uniquely on each object so for example a ```/head``` item can just check if ```slot == slot_head``` and continue on, I implemented my system nice and generically at ```/clothing``` (for ease of use, this way you just define a list of vars to change and their values and it will all just -work-) but because of that genericness I can't simply check if slot is correct (clothing has NO concept of what slot it goes it, but it does know what slot FLAGS it can go in)

tl;dr it's the nicest workaround bar actually fixing the slot system, and good luck getting anyone to actually do that.